docs/integrations/builtin/core-nodes/n8n-nodes-base.scheduletrigger.md

```diff
@@ -22,13 +22,18 @@ Add **Trigger Rules** to determine when the trigger should run.
 
 Use the **Trigger Interval** to select the time interval unit of measure to schedule the trigger for. All other rule parameters depend on the interval you select. Choose from:
 
-* [**Seconds**](#seconds-trigger-interval)
-* [**Minutes**](#minutes-trigger-interval)
-* [**Hours**](#hours-trigger-interval)
-* [**Days**](#days-trigger-interval)
-* [**Weeks**](#weeks-trigger-interval)
-* [**Months**](#months-trigger-interval)
-* [**Custom (Cron)**](#custom-cron-interval)
+- [Schedule trigger](#schedule-trigger)
+  - [Node parameters](#node-parameters)
```

Comment:
This just links to the section that we're already in, so I'm not sure it's needed.